### DISCUSSION TITLE
'Beast mode' qualifiers

### DISCUSSION THREAD

**Discussion by mavenik** [mavenik/example-app#6] [Show and tell]
![Screenshot from 2023-11-27 07-47-48](https://github.com/mavenik/example-app/assets/717284/5ecb0101-4e52-473e-83b5-a82c1fba6c46)

### It's time to go beast mode again before we conclude this course!

Every effort, sweat, and all that midnight oil that we have burnt to make things happen started attaining fruition as we saw how different pieces of code came together when we implemented a basic Playlist page

![Screenshot from 2023-11-27 08-08-06](https://github.com/mavenik/example-app/assets/717284/3c701ae9-6c2f-4870-8817-7681ebcdc72e)

 For what's coming, anybody who would like to continue in beast mode will:

1. Mark their calendars and get into intense coding sessions on Friday, Dec 1 2023 and Saturday, Dec 2 2023
2. Learn concepts related to dependency injection, database entity relationships, building forms with Laravel, applying validations and database constraints, implementing model scopes, and so on
3. Use Github's Discussions and Project management workflows with Pull Requests and Issue tracking to practice some [SDLC](https://en.wikipedia.org/wiki/Systems_development_life_cycle) concepts while performing tasks
4. Have a lot of fun and banter

### To nominate yourself, you need to do the following *before 10:00am on Tuesday, Nov 28 2023*

1. Add *mavenik* as a collaborator to your repository that contains commit history related to the Playlist page shown above. This will enable access so that I could assign tasks directly into your repository. You can do it from your repository's Settings->Collaborators section
![Screenshot from 2023-11-27 08-18-34](https://github.com/mavenik/example-app/assets/717284/3553c694-419c-447f-9d55-1fd11577e544)
2. Post a comment on this discussion with your repository's link (I will post a sample comment for your reference)

### Let's nerd out with Laravel this week!

**Comment by mavenik** [comment]
_Sample comment for nomination, you can write your own_

Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/mavenik/example-app/commits

**Reply by pravin435916** [reply]
Can't wait to go beast mode this week!
Welcoming @mavenik  !! As a collaborator on my repository.Here's my commit history for reference:
https://github.com/pravin435916/Php_code/commits

**Reply by pravin435916** [reply]
hoodie size : XL

**Comment by aniketraut16** [comment]
Can't wait to go beast mode this week!
Welcoming @mavenik !! As a collaborator on my repository.Here's my commit history for reference:
[https://github.com/aniketraut16/mylaravelapp/commits/](https://github.com/aniketraut16/mylaravelapp/commits/)

**Reply by aniketraut16** [reply]
My Hoodie Size : M (UK)

**Comment by Ayush-Sachan01** [comment]
Excited to unleash the beast mode this week! I added @mavenik , who's now on board as a collaborator for my repository. Check out the commit history here: https://github.com/Ayush-Sachan01/example-app/commits/main

**Reply by mavenik** [reply]
@Ayush-Sachan01 Woohoo!

Can't find changes to your `routes/web.php` in your main branch though. Can you check?

**Reply by Ayush-Sachan01** [reply]
@mavenik Yeah Sir, I think I fixed it now. I forget to do the second last commit yesterday.

**Reply by mavenik** [reply]
@Ayush-Sachan01 Commit messages are messed up but that's okay. Can see all pieces of code checked in. 🙌

**Reply by Ayush-Sachan01** [reply]
Hoodie size: Large(L) UK

**Comment by divyansh-2005** [comment]
Can't wait to go beast mode this week!
Added @mavenik !! As a collaborator on my repository. Here's my commit history for reference:
https://github.com/divyansh-2005/example-app/commits

**Reply by divyansh-2005** [reply]
Hoodie size: Large  L (UK)

**Comment by adi-tyasingh** [comment]


**Reply by mavenik** [reply]
@adi-tyasingh But where is the code? 😜

**Reply by adi-tyasingh** [reply]
Finally pushed all commits  :)

Hoodie Size: XL (UK)

**Comment by TheLearningHead** [comment]
Can't wait to go beast mode this week!
I welcome @mavenik !! as a collaborator on my repository. Here's the commit history for reference:
https://github.com/TheLearningHead/example-app/commits/

**Reply by TheLearningHead** [reply]
Hoodie size : L (UK)

**Comment by Samruddhi-22** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/Samruddhi-22/example-app/commits

**Reply by Samruddhi-22** [reply]
Hoodie Size-XL

**Comment by bhumimah04** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/bhumimah04/example-app/commits

**Reply by bhumimah04** [reply]
Hoodie size - Small

**Comment by Prathamesh-1011** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/Prathamesh-1011/LarvelApp/commits

**Reply by Prathamesh-1011** [reply]
Hoodie Size : XL

**Comment by Janvisahu31** [comment]
Hey there everyone....cannot wait to join you allfor next session on beast mode....I have added @mavenik as a collaborator on my repository. Check out my commit history.
https://github.com/Janvisahu31/example-app/commits

**Reply by Janvisahu31** [reply]
hoddie size:  XL

**Comment by ArpitAgrawal001** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/ArpitAgrawal001/example-app/commits

**Reply by ArpitAgrawal001** [reply]
Hoodie size: Medium(M) UK

**Comment by atharvarakshak** [comment]
Can't wait to go beast mode this week!
I just added @mavenik as a collaborator on my repository. Here's the commit history for reference: https://github.com/atharvarakshak/example-app/commits

**Reply by atharvarakshak** [reply]
Hoodie size: Medium(M) UK

**Comment by Tanmay-say** [comment]
Can't wait to go beast mode this week!
Welcoming @mavenik !! As a collaborator on my repository.Here's my commit history for reference:
[https://github.com/Tanmay-say/example-app/commits](https://github.com/Tanmay-say/example-app/commits)

**Reply by Tanmay-say** [reply]
Hoodie size : XL  (UK)

**Comment by tanmay-37** [comment]
This week is all about unleashing "beast mode"! 
I've added @mavenik  a collaborator on my GitHub repo. Check out the commit history here,
'https://github.com/tanmay-37/example-app/commits', to see the progress.

**Reply by tanmay-37** [reply]
Hoodie size : Large(L) UK

**Comment by jagritirsharma** [comment]
Excited to tackle the challenges this week with full force!

I've just added @mavenik as a collaborator to my repository. Here's the commit history for reference: 
https://github.com/jagritirsharma/example-app/commits/main

**Comment by SakshiParate03** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/SakshiParate03/example-app/commits

**Reply by SakshiParate03** [reply]
Hoodie Size: Small

**Comment by nikunjchhatani** [comment]
I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/nikunjchhatani/example-app/commits/main

**Reply by nikunjchhatani** [reply]
Hoodie size : M

**Comment by Rajdeep-Chakraborty0611** [comment]
just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
 https://github.com/Rajdeep-Chakraborty0611/example-app2/commits/main
 
 Hoodie Size : L

**Comment by Spy13x8** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference: 
https://github.com/Spy13x8/example-app/commits/main

**Reply by Spy13x8** [reply]
Hoodie Size: 2XL (UK)

**Comment by Lahotiharsh** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/Lahotiharsh/example-app/commits

**Reply by Lahotiharsh** [reply]
Hoodie size: XL

**Comment by harshshrivastava15** [comment]
Can't wait to go beast mode this week!
I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/harshshrivastava15/example-app

**Reply by harshshrivastava15** [reply]
Hoodie size: M

**Comment by adi-tyasingh** [comment]
Can't wait to go beast mode this week!
I just added @mavenik as a collaborator on my repository. Here's the commit history for reference: https://github.com/adi-tyasingh/example-app/commits/master

Hoodie Size: XL (UK)

**Comment by AmanPatne** [comment]
Can't wait to go beast mode this week!
I just added @mavenik  as a collaborator on my repository. Here's the commit history for reference:
https://github.com/AmanPatne/example-app/commits

**Reply by AmanPatne** [reply]
Hoodie Size: XL (UK)

**Comment by sanchitpahurkar** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
[https://github.com/sanchitpahurkar/example-app/commits/](url)

**Reply by sanchitpahurkar** [reply]
Hoodie Size : Medium (M)

**Comment by anu4619** [comment]
Beast mode ON!!
I just added @mavenik  as a collaborator on my repository. Here's the commit history for reference:
https://github.com/anu4619/laravel-app/commits

**Comment by Bhavikaa324** [comment]
Beast mode ON!!
I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/Bhavikaa324/laravel-app/commits

**Comment by Girish1845** [comment]
Beast mode ON!!
I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/girishnasre246/laravel-app/commits

**Comment by Arnav-28** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/Arnav-28/example-app/commits

Hoodie size : L

**Comment by javeriishika** [comment]
I just added @mavenik as a collaborator on my repository.Here's the commit history for reference:
https://github.com/javeriishika/example-app/commits

**Comment by nomaan-g** [comment]
Excited to unleash the beast mode this week! I added @mavenik , who's now on board as a collaborator for my repository. Check out the commit history here: 
https://github.com/nomaan-g/example-app/commits

Hoodie Size:  XXL

**Comment by Riddhi1444** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/Riddhi1444/example-app/commits/main

**Reply by Riddhi1444** [reply]
Hoodie size:L

**Comment by 123harshitaagrawal** [comment]
Can't wait to go beast mode this week

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/123harshitaagrawal/Honors-web-development-/commits/main

**Comment by sankezzz** [comment]
Can't wait to go beast mode this week!

I just added @mavenik as a collaborator on my repository. Here's the commit history for reference:
https://github.com/sankezzz/example-app/commits/main

**Reply by sankezzz** [reply]
hoodie size (M)

**Comment by mavenik** [comment]
Great work, beasts!

Closing discussion at deadline. Will be reviewing code and finalizing a list of qualifiers today.

💪✨
